frontend/www/js/omegaup/login/sign_in.ts

```diff
@@ -0,0 +1,91 @@
+import { OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import * as api from '../api';
+import * as UI from '../ui';
+import T from '../lang';
+import Vue from 'vue';
+import login_SignIn from '../components/login/SignIn.vue';
+import VueRecaptcha from 'vue-recaptcha';
+
+OmegaUp.on('ready', () => {
+  const payload = JSON.parse(
+    (<HTMLElement>document.getElementById('payload')).innerText,
+  );
+
+  let signIn = new Vue({
```

Comment:
```suggestion
  const signIn = new Vue({
```

hay que intentar usar `const` siempre que se pueda.